src/main/java/com/bob/jr/TextToSpeech/TextToSpeech.java

```diff
@@ -14,12 +18,52 @@
 
 public class TextToSpeech {
 
-    private static final Map<Member, MemberVoiceConfig> memberVoiceMap = new HashMap<>();
+    private static final Map<Snowflake, MemberVoiceConfig> memberVoiceMap = new HashMap<>();
+    private static final String NEARLINE_BOBJR_SETTINGS = "nearline-bobjr-settings";
+    private static final String MEMBER_VOICE_SETTINGS_FOLDER = "member-voice-settings";
     private final TextToSpeechClient textToSpeechClient;
     private final Logger logger = LoggerFactory.getLogger(TextToSpeech.class);
 
+    private final Optional<Storage> gStorageOptional;
+
     public TextToSpeech() throws IOException {
+        this(null);
+    }
+
+    public TextToSpeech(Storage gStorage) throws IOException {
         textToSpeechClient = TextToSpeechClient.create();
+        final var loadFromStorage = true;
```

Comment:
Brother can you move this out of the constructor and into a function - sheeeeee

Reply:
Yes